pkg/test-infra/pkg/tidb/recommend.go

```diff
@@ -58,7 +61,20 @@ func (t *TiDBClusterRecommendation) TiDBReplicas(replicas int32) *TiDBClusterRec
 	return t
 }
 
-// tidb-operator do not have sane defaults yet, do a recommendation
+func buildImage(name string) string {
```

Comment:
`buildImage`  is ambiguity...

Reply:
Do you have some advice for the function name?